catalog/templates/catalog/course.html

```diff
@@ -2,7 +2,7 @@
 
 {% load static %}
 {% load humanize %}
-{% load has_perm_on %}
+{% load has_write_perm_on %}
```

Comment:
I would prefer a single file to load (maybe plainly `{% load permissions %}`) that exposes 2 templatetags: `has_write_perm_on ` and `has_moderation_perm_on `

Reply:
I tried but got issue with two tags on the same file so I assume it need one by file . Will make more search on that